share/picongpu/examples/LandauDamping/include/picongpu/param/physicalConstants.param

```diff
@@ -0,0 +1,108 @@
+/* Copyright 2013-2018 Axel Huebl, Heiko Burau, Rene Widera, Richard Pausch,
+ *                     Marco Garten
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+
+#pragma once
+
+namespace picongpu
+{
+    constexpr float_64 PI = 3.141592653589793238462643383279502884197169399;
+
+    namespace SI
+    {
+        /** unit: m / s */
+        constexpr float_64 SPEED_OF_LIGHT_SI = 1.0;
+
+        /** unit: N / A^2 */
+        constexpr float_64 MUE0_SI = PI * 4.e-7;
+        /** unit: C / (V m) */
+        constexpr float_64 EPS0_SI = 1.0 / MUE0_SI / SPEED_OF_LIGHT_SI
+            / SPEED_OF_LIGHT_SI;
+
+        /** reduced Planck constant
+         * unit: J * s
+         */
+        constexpr float_64 HBAR_SI = 1.054571800e-34;
+
+        // Electron properties
+        /** unit: kg */
+        constexpr float_64 ELECTRON_MASS_SI = 9.109382e-31;
+        /** unit: C */
+        constexpr float_64 ELECTRON_CHARGE_SI = -1.602176e-19;
+
+        /* atomic unit for energy:
+         * 1 Rydberg = 27.21 eV --> converted to Joule
+         */
+        constexpr float_64 ATOMIC_UNIT_ENERGY = 4.36e-18;
+
+        /* atomic unit for electric field in V/m:
+         * field strength between electron and core in ground state hydrogen
+         */
+        constexpr float_64 ATOMIC_UNIT_EFIELD = 5.14e11;
+
+        /* atomic unit for time in s:
+         * 150 attoseconds (classical electron orbit time in hydrogen)  / 2 PI
+         */
+        constexpr float_64 ATOMIC_UNIT_TIME = 2.4189e-17;
+
+        /** Avogadro number
+         * unit: mol^-1
+         *
+         * Y. Azuma et al. Improved measurement results for the Avogadro
+         * constant using a 28-Si-enriched crystal, Metrologie 52, 2015, 360-375
+         * doi:10.1088/0026-1394/52/2/360
+         */
+        constexpr float_64 N_AVOGADRO = 6.02214076e23;
+    }
+
+    // converts
```

Comment:
only a note please use `/* ... */ for comments, maybe this is already wrong in our default setting

Reply:
this is taken from the default and needs adjustment